### ISSUE TITLE
Ampersand and Apostrophe

### ISSUE THREAD

**Issue by klou** [yathit/sugarcrm-gmail-chrome-extension#2]
Just ran across this and trying it out with my local version of SugarCRM.

Apostrophes in Account Names (and elsewhere) seem to be rendered as &#039;

I'm assuming similar issues for ampersand.

So far, good work!

**Comment by klou** [comment]
Sorry, that wasn't correct.
```
Apostrophe's in the data are being rendered as &#039s;
```

**Comment by yathit** [comment]
Thanks for the report.

Could you show me screenshoot of the uncorrect rendering. I want to see where it is.

Kyaw

**Comment by yathit** [comment]
Thanks, I got it. It will be fix in next release.

**Comment by klou** [comment]
This is obviously related - but I'll just add it here. 
```
Double-quotes in the text are rendered as &quot;
```

**Comment by yathit** [comment]
Fixed in rel 3.1.2.

![cdj1eejko9](https://cloud.githubusercontent.com/assets/2137932/6860773/b3a980dc-d467-11e4-8096-508a6c4bdf7b.gif)

**Comment by yathit** [comment]
![3tya62anjn](https://cloud.githubusercontent.com/assets/2137932/6860849/ab349558-d468-11e4-99b0-ffb29992d898.gif)
